### DISCUSSION TITLE
Add the ability to delete a note without a prompt

### DISCUSSION THREAD

**Discussion by cuprum** [ahyatt/ekg#52] [Ideas]
For those who know what they are doing (like me 😊). With `C-u`, that is `C-u d` in the notes buffer.

It will also be possible to assign custom keybinding for quick deletion.

I can make a PR.

**Comment by ahyatt** [comment]
Good idea, yes, can you send a PR?

**Comment by cuprum** [comment]
Done.

**Reply by jayrajput** [reply]
Interested to see the PR. I am interested to probably introduce a new variable like `ibuffer-experts` to not ask for confirmation for dangerous operations. Anyway the notes are in the trash, so can be restored. Also there are multiple backups.

**Reply by ahyatt** [reply]
https://github.com/ahyatt/ekg/pull/53 is the pull request.  I agree that a variable is a possibility.  However, another possibility is just to never warn, since, as you say, it goes in the trash.  I think right now it's good enough that there's not a strong incentive to change, but I think as we use ekg more it might be more clear what an optimal solution is.

**Reply by jayrajput** [reply]
Sure. I agree. Thanks.
